axon-server-connector/src/test/java/org/axonframework/axonserver/connector/query/StreamingQueryEndToEndTest.java

```diff
@@ -141,6 +146,14 @@ private AxonServerQueryBus axonServerQueryBus(SimpleQueryBus localSegment, Strin
                                  .updateEmitter(emitter)
                                  .genericSerializer(serializer)
                                  .messageSerializer(serializer)
+                                 .executorServiceBuilder((axonServerConfiguration, runnables) ->
```

Comment:
Tests should work even without customizing the `Executor`?

Reply:
Yes, but reducing the number of threads to 1 (compared to the default of 10) would make the tests more reliably failing in case of issues I think

Reply:
Reverting this, seems it doesn't matter